### ISSUE TITLE
RFC: What Type of Package Should the Key Master Install As?

### ISSUE THREAD

**Issue by ahoefling** [HoeflingSoftware/Dnn.KeyMaster#52]
Currently the Key Master installs as a Persona Bar Extension but it is more than just that

* Data Provider
* SQL Membership Provider
* Connects with azure key vault using a specified configuration
* Persona Bar UI
  * Controls Azure Config
  * Controls App Settings Config
  * Future - will control other settings as added

Currently the extension installs as 1 Persona Bar extension. Would this be better marked under another category such as:

* Providers
* Connectors

**Comment by mitchelsellers** [comment]
This is a tricky one, however, I think that a PB Extension is the most easily identifiable.

The actual extension type doesn't really matter, except for discoverability.  The most visible in this case is the PB interface.

**Comment by nvisionative** [comment]
I agree with PersonaBar extension label and approach- makes the most sense to me.
